Explorer/Assets/DCL/SDKComponents/Tween/Systems/TweenLoaderSystem.cs

```diff
@@ -47,9 +45,11 @@ private void UpdateTween(ref PBTween pbTween, ref SDKTweenComponent tweenCompone
         {
             if (pbTween.ModeCase == PBTween.ModeOneofCase.None) return;
 
-            if (pbTween.IsDirty || !TweenSDKComponentHelper.AreSameModels(pbTween, tweenComponent.CurrentTweenModel))
+            // (Juani & Fran): Im not happy to leaave this AreSameModels check. But apparently its required as SDK might not mark the tween component as dirty.
+            // Its present in the old renderer. If this was not needed, the CurrentTween field can be deleted
+            if (pbTween.IsDirty || !TweenSDKComponentHelper.AreSameModels(pbTween, tweenComponent.CachedTween))
             {
-                tweenComponent.CurrentTweenModel = new SDKTweenModel(pbTween);
+                tweenComponent.CachedTween = pbTween;
```

Comment:
You should not cache PB Components as their life-cycle is fully under control of the CRDT bridge. The previous reference could be already re-utilized for another entity. If it works it's a happy coincidence.

You have to copy values if you want to compare them

Reply:
Indeed. I copied them when I created it, but here I just assigned the reference. Fixed!